src/components/FloatingLabel/FloatingLabel.tsx

```diff
@@ -7,9 +7,9 @@ import type { DeepPartial } from '../../types';
 import type { FlowbiteSizes } from '../Flowbite';
 
 export interface FlowbiteFloatingLabelTheme {
-  input: any;
-  label: any;
-  helperText: any;
+  input: any; // eslint-disable-line
+  label: any; // eslint-disable-line
+  helperText: any; // eslint-disable-line
```

Comment:
This is probably: FlowbiteColors

Reply:
witch one do you mean @rluders?

Reply:
Sorry, the `helperText` is the `FlowbiteColors` type. :)

Reply:
@rluders 
the type `FlowbiteColors ` is like this:

```
export interface FlowbiteColors extends FlowbiteStateColors {
  [key: string]: string;
  blue: string;
  cyan: string;
  dark: string;
  gray: string;
  green: string;
  indigo: string;
  light: string;
  lime: string;
  pink: string;
  purple: string;
  red: string;
  teal: string;
  yellow: string;
}
```

while the `helperText` is used like this:
```
  helperText: {
    default: 'mt-2 text-xs text-gray-600 dark:text-gray-400',
    success: 'mt-2 text-xs text-green-600 dark:text-green-400',
    error: 'mt-2 text-xs text-red-600 dark:text-red-400',
  },
```

this will cause error because colors are not optional in `FlowbiteColors`

Reply:
maybe

Reply:
I think that it would be fine to use `FlowbiteColors`. It extends the `FlowbiteStateColors` that has all the states, except by the `default`, and I don't think that it would be a problem.

Reply:
@rluders 
![image](https://github.com/themesberg/flowbite-react/assets/66066475/376124b2-8e3b-4557-a4f7-8c457ba9305b)
Are you sure this is OK?

Reply:
Did you change the https://github.com/themesberg/flowbite-react/blob/0f7ca75d5709eded9b9a1c9e406fc28962aba7ad/src/components/FloatingLabel/FloatingLabel.tsx#L33C1-L33C18 to be:

```
helperText: keyof FlowbiteColors;
```

??

Reply:
(BTW: Not saying that this PR is going to solve it, but we need to put all the colors into some standard interface - the colors patterns are becoming really complicated to manage)

Reply:
> keyof

after doing that, this is the result:
![image](https://github.com/themesberg/flowbite-react/assets/66066475/8f3f6912-6068-411f-972d-f49a726e91f7)

Reply:
@paghar if necessary you can create a specific type for the `FlowbiteFloatingLabelHelperText` (oh my... look at this name) extending the `FlowbiteColors` and picking only the ones that you want. There are some components (Button?) that do something similar.